smart_contracts/contract/src/lib.rs

```diff
@@ -56,7 +56,6 @@
     html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
     test(attr(forbid(warnings)))
 )]
-#![warn(missing_docs)]
```

Comment:
Removed by mistake?

Reply:
Addressed in https://github.com/casper-network/casper-node/pull/1854/commits/1f3d894b65808058e51fb5884e0e6e4674556047